manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FunctionsTest.scala

```diff
@@ -122,7 +122,7 @@ class FunctionsTest extends DocumentingTestBase {
 
   @Test def size() {
     testThis(
-      title = "size()",
+      title = "size() - list",
```

Comment:
I believe the title controls the file name. With a title of "size() - list" the file name becomes "size---list.asciidoc", which means the include directive for this content fails. If you run a `find manual/cypher/cypher-docs/ -name "*size*"` you should see the generated files. Mutatis mutandis also the other "size" and the two "length" items.